docs/packages/pkg/Qwt.rst

```diff
@@ -0,0 +1,21 @@
+.. spelling::
+
+    Qwt
+
+.. index::
+  single: unsorted ; Qwt
+
+.. _pkg.Qwt:
+
+Qwt
+===
+
+-  `Official <svn.code.sf.net/p/qwt/code/branches/qwt-6.1>`__
+-  `Hunterized <https://github.com/hunter-packages/Qwt>`__
```

Comment:
There is no such repository, do you want me to create it?

Reply:
Yes, need tag 6.1.3